assets/pages/regular-contributions/components/regularContributionsPayment.jsx

```diff
@@ -104,6 +106,7 @@ function RegularContributionsPayment(props: PropTypes, context) {
       props.contributionType,
       props.dispatch,
       'stripeToken',
+      'Stripe',
```

Comment:
We already have the `PaymentFieldName`, that could be used to proxy this logic. Is it possible to delete one of the two fields?

Reply:
The best probably is to delete `PaymentFieldName` from the argument list and then inside the function have a dictionary which maps from `PaymentMethod` to `PaymentFieldName`.